xarray/tests/test_dataset.py

```diff
@@ -4,7 +4,6 @@
 from collections import OrderedDict
 from copy import copy, deepcopy
 from io import StringIO
-import pickle
```

Comment:
I think we still need this -- there's a pickle test in this file that is failing on Travis-CI now.